assets/src/components/ui/image/avatar.js

```diff
@@ -42,13 +42,17 @@ export class AvatarImage extends Component {
 			avatarWidth,
 			avatarAltText,
 		} = this.props;
+
+		const style = {
+			height: avatarHeight,
+			width: avatarWidth,
+		};
 		return avatarUrl ? (
 			<div className={ avatarClass + '-image-wrap-div' }>
 				<img
 					className={ avatarClass + '-avatar-img avatar' }
 					src={ avatarUrl }
-					height={ avatarHeight }
-					width={ avatarWidth }
+					style={ style }
```

Comment:
Why style instead of height/width?  Unfortunately if any css selectors that match this img tag have `height` or `width` set they automatically override any height and width html attributes on the html tag (the default height/width is "cached" as the original size of the img). In order to have the img size changes seen immediately in the editor, we need to use style tags. 

For server rendering on the frontend I _do_ use the height and width html attributes because the size of the image isn't changing at all after it loads.  So that still allows clients to control the size via css if they want on the frontend.